### DISCUSSION TITLE
Nothing displays in page 2.

### DISCUSSION THREAD

**Discussion by magic-thomas** [filamentphp/filament#6932] [Help] [answered]
I just made PostResource with --generate option. 

There are data list in database. so it shows page numbers on the list page. 

When I click page 2, or 3 etc, it shows nothing. 

And there is error on dev tools. 

I didn't change anything in PostResource and subfolder. 

How can I fix this ? 

I am using Laravel 10.x ( latest ) / livewire 2.x ( latest ) . 

<img width="892" alt="Screen Shot 2023-07-06 at 4 09 00 AM" src="https://github.com/filamentphp/filament/assets/69987486/781c98ea-babe-4c67-8562-b6cfff1ed262">
<img width="887" alt="Screen Shot 2023-07-06 at 4 06 50 AM" src="https://github.com/filamentphp/filament/assets/69987486/66bb35db-5f07-4853-b371-8b0795635f77">
<img width="877" alt="Screen Shot 2023-07-06 at 4 06 25 AM" src="https://github.com/filamentphp/filament/assets/69987486/d0b599ab-b101-4fa0-a5c0-88956d0a7a68">

The error / warning in dev tools are seen in page 1 also.

**Comment by danharrin** [comment]
Very weird, but we can't really debug this without a minimal reproduction repository. Please open an issue with it.

**Reply by magic-thomas** [reply]
Sorry but I can not upload my whole projects. 

I did this for figuring out the problem. 

1. I removed app/Filament/Resources folder. 

2. Remade it by 'php artisan make:filament-resource PostResource   --generate'


And I accessed again 'http://127.0.0.1:8000/admin/posts'. 

There are errors again. 

I thought Filament package could be used independently from old projects. 

But it seems not.

Thanks for your corcenr.

**Reply by danharrin** [reply]
I'm not asking you to upload your whole project, you should reproduce the error very simply in a new project. Read the instructions we provide on the New Issue form.

**Reply by magic-thomas** [reply, marked as answer]
Sorry for my somehow rudeness. Not intended to hurt someone or this great package. 

I am  the first and the only guy who commented in Reddit at the very first stage of this package. 

Anyway I found why. 

'spatie/laravel-blade-comments' package conflicts with filament package. 

Even setting 'spatie/laravel-blade-comments' disabled in config, it conflicts with filament. 

I removed 'spatie/laravel-blade-comments' for admin panel. 

Thanks.

**Reply by danharrin** [reply]
It didn't come across as rude, don't worry, you just misunderstood what I meant by a reproduction repository
